modules/meeting/app/components/meetings/row_component.rb

```diff
@@ -47,7 +47,24 @@ def duration
     end
 
     def location
-      h(model.location)
+      if valid_meeting_location_link?
```

Comment:
We already use riku for automatically linking to URLs. I think we could just reuse this here? https://github.com/opf/openproject/blob/dev/lib/open_project/text_formatting/filters/autolink_filter.rb#L46

`